zulipterminal/ui_tools/boxes.py

```diff
@@ -1361,6 +1361,30 @@ def main_view(self) -> List[Any]:
         ]
         return [part for part, condition in parts if condition]
 
+    def update_msg_content_header(self) -> bool:
+        """
+        Update the content header of a message box (if required).
+        """
+        update_required = False
+        content_header = None
+
+        # Get message content header (if present)
+        if self.need_recipient_header():
+            content_header = self[1][1]
+        elif isinstance(self[0][1], urwid.Text):
+            content_header = self[0][1]
+
+        # Update the content header only if it's present and non-empty.
+        if content_header is not None:
+            update_required = content_header.text != " "
```

Comment:
It's not obvious where the `" "` test comes from - it's the default author text, when there's no author in the header, right? That's partly unclear due to the content_header variable name, but "non empty" and "content header" just repeat the code, so if we're going to include a comment, let's make it more useful - or set a variable like `author_is_present = ...` to help.

Reply:
Makes sense. I'll go with `author_is_present` and skip the comment :+1: